code/modules/paperwork/photocopier.dm

```diff
@@ -79,8 +79,7 @@
 		updateUsrDialog()
 	else if(href_list["remove"])
 		if(copyitem)
-			copyitem.loc = usr.loc
-			usr.put_in_hands(copyitem)
+			copyitem.forceMove(get_turf(src))
```

Comment:
Giving an item to a person/ejecting it is a fairly common action of a machine - like the sleepers(beakers), cryotubes(beakers), R&D console(design/tech disks)... I suppose I can PR up that change myself, however